sdk/opa.go

```diff
@@ -237,6 +239,11 @@ func (opa *OPA) Decision(ctx context.Context, options DecisionOptions) (*Decisio
 		}
 	}
 
+	var p topdown.QueryTracer
+	if options.Profiler != nil {
+		p = options.Profiler
+	}
```

Comment:
[golf] Is this different from
```go
p := options.Profiler
```
or, if the type wrangling is intended,
```go
var p topdown.QueryTracer = options.Profiler
```
? Just since if `options.Profiler` actually is `nil`, `p` would end up still being `nil`, too, wouldn't it? 🤔

Reply:
Hmm, yes... 😊 

This refactors away quite nicely as part of b0aade94f32d7937bcb4319618f00b864e67db29